### ISSUE TITLE
How to trace Unable to resolve link; aborting: ID_LINK

### ISSUE THREAD

**Issue by RidaAyed** [kaushalmodi/ox-hugo#456]
I can't export the buffer nor any subtree. Grepping the org file for `ID_LINK` returns nothing. Where should I start debugging?

Updating ox-hugo didn't change anything.

**Comment by kaushalmodi** [comment]
@RidaAyed Is this a recent regression in behavior after ox-hugo update?

Can you provide a minimal Org file example where this issue can be reproduced?

**Comment by RidaAyed** [comment]
> Is this a recent regression in behavior after ox-hugo update

No. I've updated and rolled back to no avail.

Grepping the complete computer finds the ID in another org file.
However I can't find the relation.

I noticed that one or more (non-blog) subtree/s generates the error. Now digging
deeper (org file contains >30k lines)

One of these subtrees generates not only that error but this
line before the actual error as well:

4 files scanned, 2 files contains IDs, and 2 IDs found. [2 times]

org-export-resolve-id-link: Unable to resolve link; aborting: "0a721ede-11d0-4ec9-ad34-c24a94807f5d"
